### ISSUE TITLE
DPL-686 Test Platform Solutions RabbitMQ platform

### ISSUE THREAD

**Issue by emrojo** [sanger/General-Backlog-Items#275]
**Description**
As a developer I want to test the new Rabbitmq setup by Platform Solutions because we want to provide a more reliable level of service for Heron and other queue related projects.

This story should:
- [x] Test we can connect via HTTPS by default
- [x] Test we can connect via HTTPS by default from an OpenStack instance
- [x] Sanity check we can send and receive messages
- [x] Record access credentials in our KeePas credentials database 

**Who the primary contacts are for this work**
Eduardo Martin
Stephen Inglis

**Knowledge or Stake holders**

**Additional context or information**
Currently PSD maintain our own RabbitMQ message bus. 
To reduce the number of servers and deployments we have to maintain we would like to switch over to a RabbitMQ message bus provided and maintained by Platform Solutions.

**Comment by SujitDey2022** [comment]
Hi @emrojo @Sangeetha-Bheeman wanted to confirm if this story is still relevant and needs to be done. Please let me know.

**Comment by javidahmed64592** [comment]
Progress update: I managed to create test queues and exchanges on the `PSD_Rabbit_Playground` VHost. I used Stuart's RabbitMQ scripts from the demo to publish and consume messages, and confirmed the messages do appear in RabbitMQ and get consumed successfully when a consumer is available.

This was done on port `5672` which is the unencrypted port. I was unable to do the same with port `5671` which is encrypted, possibly requiring SSL to be set up on the server.
